### PR TITLE
[ENDPOINT] GET /slides/:id OT246-58

### PR DESCRIPTION
**Pull request by kamp3r**
AS user admin
I WANT to see slides detail
TO see contain detail

Approbation criteria:

- Make a get request to /slides/;id
- It will show all the fields of the slide based on the id sent by parameter.
- In the event that it does not exist, it will return an error with a 404 status code

### Evidence

test1

test2

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 
 app.use('/', indexRouter)
+app.use('/slides', slidesRouter)
```
Comment (gabodmc) [inline app.js]: El slidesRouter tiene que ser llamado adentro del indexrouter. Al app.js no lo modificamos.

**Review by gabodmc — CHANGES_REQUESTED** [review]
Lo veo muy bien, solo unos cambios

File: app.js (hunk 1 above)
Comment (kamp3r) [reply app.js]: Realizado!

**Review by gabodmc — APPROVED** [review]

**Review by Samuel0023 — APPROVED** [review]

**Review by bgalacho — APPROVED** [review]
Muy bueno Martin!

**Pull request merged**